README.md

```diff
@@ -44,23 +44,49 @@ Set your different credentials in the file by uncommenting the lines and adding
 
 ## Usage
 
+This show you an example of a settings
+
+```ruby
+AppleSignIn.configure do |config|
+  config.apple_client_id = 'com.yourapp...'
+  config.apple_private_key = "-----BEGIN PRIVATE KEY-----\nMIGTAgEA....\n-----END PRIVATE KEY-----"
```

Comment:
SHouldn't we recommend to use an ENV variable here?

Reply:
Good advice, i will add a message to recommend it 😄